ext/RMagick/extconf.rb

```diff
@@ -84,21 +86,51 @@ def configure_compile_options
           end
         end
 
-        # either set flags using Magick-config or pkg-config (new Debian default)
-        if $magick_config
-          # Save flags
-          $CFLAGS     = ENV['CFLAGS'].to_s   + ' ' + `Magick-config --cflags`.chomp
-          $CPPFLAGS   = ENV['CPPFLAGS'].to_s + ' ' + `Magick-config --cppflags`.chomp
-          $LDFLAGS    = ENV['LDFLAGS'].to_s  + ' ' + `Magick-config --ldflags`.chomp
-          $LOCAL_LIBS = ENV['LIBS'].to_s     + ' ' + `Magick-config --libs`.chomp
+        # From ImageMagick 6.9 binaries are split to two and we have to use
+        # MagickWand instead of MagickCore
+        checking_for('newer version of ImageMagick (>= 6.9)') do
+          version_data = $magick_version.split('.').map(&:to_i)
+          $with_magick_wand = version_data[0] > 6 || (version_data[0] == 6 && version_data[1] > 8)
```

Comment:
Sorry I did not make myself clear. I mean, instead of splitting version and comparing integers, could you create `Gem::Version` objects and compare them, like we do in gemspec? It will be easier to read :) Everything else is fine and you did a good job!